File: .sdk/tm/ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { AbdoStoreSDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await AbdoStoreSDK.test()
    equal(null !== testsdk, true)
  })

})

File: lua/test/exists_test.lua
-- AbdoStore SDK exists test

local sdk = require("abdo-store_sdk")

describe("AbdoStoreSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
